scripts/createbooktype

```diff
@@ -720,6 +725,10 @@ if __name__ == '__main__':
         if '-' in project_name:
             raise MajorError("Minus not allowed in project name. Try to use '_' instead.")
 
+        if args.virtual_env:
+            activate_this = '{}/bin/activate_this.py'.format(args.virtual_env)
```

Comment:
@aerkalov  it is required to active the env, otherwise the commands below could fail if libraries are not present in global environment
check_django_version
check_lxml_available
check_feedparser_available
check_pil_available
check_redis_available
check_south_available
check_unidecode_available